frontend/src/metabase-types/api/mocks/actions.ts

```diff
@@ -34,6 +34,7 @@ export const createMockQueryAction = ({
 }: Partial<WritebackQueryAction> = {}): WritebackQueryAction => {
   return {
     id: 1,
+    entity_id: "ACTIONmetabaseACTIONs",
```

Comment:
Should we extract this to a constant for easy testing of entity ids, similar to `ORDER_ID` we have in e2e helpers? My guess is that this weird format in `ACTIONmetabaseACTIONs ` is the format of entity ids

Reply:
yeah that's fair. I was feeling a little silly typings these 😅

Reply:
Ok, I made a `createMockEntityId` helper and tightened `BaseEntityId`